src/main.rs

```diff
@@ -261,6 +311,43 @@ fn create_pipeline(app: &App) -> Result<(gst::Pipeline, gtk::Widget), Box<dyn er
                 inner.error = Some(Box::new(err.get_error()));
                 inner.application.quit();
             }
+            MessageView::Element(msg) => {
+                // Catch the EOS messages from our filesink. Because the other sink,
```

Comment:
What is EOS?

Reply:
Ah I wrote it out elsewhere but forgot to do so here. Thanks :) End Of Stream